pkg/operator/ceph/cluster/mon/spec.go

```diff
@@ -49,8 +51,11 @@ func (c *Cluster) getLabels(monConfig *monConfig, canary, includeNewLabels bool)
 		labels["mon_canary"] = "true"
 	}
 	if includeNewLabels {
-		if c.monVolumeClaimTemplate(monConfig) != nil {
+		monVolumeClaimTemplate := c.monVolumeClaimTemplate(monConfig)
+		if monVolumeClaimTemplate != nil {
+			size := monVolumeClaimTemplate.Spec.Resources.Requests[v1.ResourceStorage]
 			labels["pvc_name"] = monConfig.ResourceName
+			labels["pvc_size"] = strconv.FormatInt(size.Value(), 10)
```

Comment:
I believe you just need to call `String()`
```suggestion
			labels["pvc_size"] = size.String()
```

Reply:
Done